src/services/quotes/source-lists/types.ts

```diff
@@ -7,6 +7,7 @@ import { QuoteRequest, SourceMetadata, SourceId, GlobalQuoteSourceConfig, QuoteT
 export type IQuoteSourceList = {
   supportedSources(): Record<SourceId, SourceMetadata>;
   getQuote(request: SourceListRequest): Promise<SourceListResponse>;
+  getQuotes(request: MultipleSourceListRequest): Promise<SourceListResponse[]>;
```

Comment:
This should be an array of promises, instead of a promise of an array. So that we can run everything in parallel later. On the `batch-api` source list, we will "fake" having things run in parallel, since it's all in one API call, but it will still be true for `api` and `local` implementations
```suggestion
  getQuotes(request: MultipleSourceListRequest): Promise<SourceListResponse>[];
```